rpc/client/dcrwallet/methods.go

```diff
@@ -701,7 +701,7 @@ func (c *Client) FundRawTransaction(ctx context.Context, rawhex string, fundAcco
 	return res, err
 }
 
-// GetInfoWallet calls the getinfo method.  It is named differently to avoid a
+// GetInfo calls the getinfo method.  It is named differently to avoid a
```

Comment:
This should read

```
// GetInfoWallet calls the getinfo method.  It is named differently to avoid a
// naming clash for dcrd clients with a GetInfo method.
func (c *Client) GetInfoWallet(ctx context.Context) (*types.InfoWalletResult, error) {
	res := new(types.InfoWalletResult)
	err := c.Call(ctx, "getinfowallet", res)
	return res, err
}
```

Reply:
Thanks for your guidance.

Modified. Please review again.

Reply:
see my edit, the actual method called should still be "getinfo"

Reply:
Sorry for the mistake. Edited.